pages/event.tsx

```diff
@@ -1,8 +1,35 @@
 import React from "react";
 import { NextPage } from "next";
+import Head from "next/head";
 
-const EventPage: NextPage = () => {
-  return <div>Event</div>;
+import { getHeadersFromContext } from "../src/utils/index";
+import { NextPageProps } from "../src/types";
+
+import { Page } from "../src/components/Layout/Page";
+import { EventContent } from "../src/components/Event/EventContent";
+
+// need fix
+export const EventPage: NextPage<NextPageProps> = ({ headers }) => {
+  return (
+    <>
+      <Head>
+        <meta property="description" content="These are media events which TradeTrust has been involved in." />
+        <meta property="og:description" content="These are media events which TradeTrust has been involved in." />
+        <meta property="og:title" content="TradeTrust - An easy way to check and verify your documents" />
```

Comment:
looks like the default title, maybe can be used as default and overridden when needed ?